src/annotator/test/sidebar-test.js

```diff
@@ -402,6 +402,28 @@ describe('Sidebar', () => {
       });
     });
 
+    describe('on "toastMessageAdded" event', () => {
+      it('re-publishes event via emitter', () => {
+        const sidebar = createSidebar();
+        sinon.stub(sidebar._emitter, 'publish');
+        emitSidebarEvent('toastMessageAdded', {});
+        assert.calledWith(sidebar._emitter.publish, 'toastMessageAdded', {});
+      });
+    });
+
+    describe('on "toastMessageDismissed" event', () => {
+      it('re-publishes event via emitter', () => {
+        const sidebar = createSidebar();
+        sinon.stub(sidebar._emitter, 'publish');
+        emitSidebarEvent('toastMessageDismissed', 'someId');
+        assert.calledWith(
+          sidebar._emitter.publish,
+          'toastMessageDismissed',
+          'someId'
+        );
+      });
+    });
```

Comment:
These tests are not entirely correct, as they are coupling with sidebar implementation details. In fact, the `_emitter` property is supposedly private, so the tests should not rely on it.

However, the rest of the events are tested like this, so I have decided to keep the same approach, and tackle this separately, as a task of its own.